portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,14 @@
  */
 function addRandomGreeting() {
   const greetings =
```

Comment:
change greetings here to something more representative of the content and keep method name and other variables consistent